app/assets/stylesheets/styles/components/crops/components/_crop_info_container.scss

```diff
@@ -31,6 +41,18 @@
   }
 }
 
+.small-crop img {
```

Comment:
@TanSA05. I think @kgrz means that you should use a `.class` instead of the the `img` tag, which is a element-level. The reason for this is that element-level CSS is often unnecessarily specific and can cause problems with debugging further down the line. I'm not sure it's a huge issue though in this case.